### PR TITLE
KFSPTS-10783 upgrade to 2017-11-30 version of financials

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/krad/service/impl/CuMaintainableXMLConversionServiceImpl.java [hunk 1]
```diff
@@ -11,5 +11,18 @@
...
         return new CuMaintenanceXMLConverter(classPropertyRuleMaps, dateRuleMap);
     }
+    
+    /**
+     * This implementation just returns the same XML that was passed in.
+     *
+     * @see org.kuali.kfs.krad.service.MaintainableXMLConversionService#transformMaintainableNoteXML(java.lang.String)
+     *
+     * @param xml
+     * @return
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/krad/service/impl/CuMaintainableXMLConversionServiceImpl.java]: Should the "param" and "return" sections be removed from this Javadoc comment?

File: src/main/java/edu/cornell/kfs/krad/service/impl/NoOpMaintainableXMLConversionServiceImpl.java [hunk 2]
```diff
@@ -16,5 +16,18 @@
...
         return xml;
     }
+    
+    /**
+     * This implementation just returns the same XML that was passed in.
+     *
+     * @see org.kuali.kfs.krad.service.MaintainableXMLConversionService#transformMaintainableNoteXML(java.lang.String)
+     *
+     * @param xml
+     * @return
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/krad/service/impl/NoOpMaintainableXMLConversionServiceImpl.java]: Another comment that could have the "param" and "return" parts cleaned up.

File: src/main/java/edu/cornell/kfs/krad/service/impl/CuMaintainableXMLConversionServiceImpl.java (hunk 1 above)
Comment (db483) [reply src/main/java/edu/cornell/kfs/krad/service/impl/CuMaintainableXMLConversionServiceImpl.java]: that makes sense, I am updating the code

File: src/main/java/edu/cornell/kfs/krad/service/impl/NoOpMaintainableXMLConversionServiceImpl.java (hunk 2 above)
Comment (db483) [reply src/main/java/edu/cornell/kfs/krad/service/impl/NoOpMaintainableXMLConversionServiceImpl.java]: that makes sense, I am updating the code

**Comment by db483** [comment]
Code has been updated. Thank you.

**Pull request merged**